app/scripts/views/mixins/password-mixin.js

```diff
@@ -39,17 +32,11 @@ define(function (require, exports, module) {
       try {
         var passwordField = this.$('.password');
         if (isVisible) {
-          passwordField.attr('type', 'text').attr('autocomplete', 'off')
-            .attr('autocorrect', 'off').attr('autocapitalize', 'off');
+          passwordField.attr('type', 'text').attr('autocorrect', 'off').attr('autocapitalize', 'off');
           this.logViewEvent('password.visible');
         } else {
-          passwordField.attr('type', 'password');
-          if (this.isPasswordAutoCompleteDisabled()) {
-            passwordField.attr('autocomplete', 'off');
-          } else {
-            passwordField.removeAttr('autocomplete')
+          passwordField.attr('type', 'password').removeAttr('autocomplete')
```

Comment:
@vladikoff I did not modify the function setPasswordVisibbilityFromButton. Do you mean rather in the setPasswordVisibility?